gr-digital/grc/digital_ofdm_sync_sc_cfb.block.yml

```diff
@@ -15,6 +15,10 @@ parameters:
     options: ['False', 'True']
     option_labels: [Odd, Even]
     hide: part
+-   id: threshold
+    label: Threshold
+    dtype: real
+    default: '0.9'
 
```

Comment:
Missing two asserts here: `fft_len > 0` and `cp_len >= 0`

From these lines: https://github.com/gnuradio/gnuradio/commit/ea468b8d1ff4283c8659487bcf4689e98d00d79e#diff-beb772f4d9d0b00937f98b136e97aef5R39

Reply:
Thanks :)